docs/overview/identity.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-
+sidebar_position: 2
```

Comment:
Manually specifying the position in each document sucks. Can we list the documents in `sidebars.js` in the desired order?